### ISSUE TITLE
Versioning on PackageControl doesn't work

### ISSUE THREAD

**Issue by naokazuterada** [naokazuterada/MarkdownTOC#123]
I noticed the 'VERSION' on [PackageControl](https://packagecontrol.io/packages/MarkdownTOC) did not match the intended version (3.0.1). Something wrong...

Problem: [This makes people not to notice release note](https://github.com/naokazuterada/MarkdownTOC/issues/122#issuecomment-374990483)

![screen shot 2018-03-23 at 1 14 17](https://user-images.githubusercontent.com/270925/37783134-094de686-2e38-11e8-99f1-4dabbf11218c.png)

**Comment by Odyseus** [comment]
Hello, @naokazuterada.

First, I would like to make clear that I'm not an expert in Package Control and I never published a package there.

Reading its documentation, you may need to set the **tags** key to **true** in [your **package_control_channel** repository](https://github.com/wbond/package_control_channel/blob/master/repository/m.json#L563).

An [excerpt of the documentation](https://packagecontrol.io/docs/submitting_a_package) reads:

> Please note that branch-based releases have been deprecated and no new packages will be accepted that utilize that feature. Existing packages are encouraged to update to using tags.

> For other options, please read the documentation in the example-repository.json file. Please note that the version, url and date sub-fields are not allowed in the centralized channel. If you wish to use these, you must host your own packages.json.

So, basically, you may need to remove `"branch": "master"` and add `"tags": true`.

**Comment by naokazuterada** [comment]
@Odyseus 
Wow! thanks for your taking time.
I will try!

**Comment by naokazuterada** [comment]
I send a PR.
https://github.com/wbond/package_control_channel/pull/7013

**Comment by naokazuterada** [comment]
@Odyseus 
Finally the versioning is fixed. Thanks for your help!
